### PR TITLE
[#43158] Impossible to deselect all days when choosing days to receiv…

### PR DESCRIPTION
**Pull request by DmitryBash**
…e mail reminders

### CODE REVIEW

File: frontend/src/app/features/user-preferences/state/user-preferences.store.ts [hunk 1]
```diff
@@ -26,7 +26,6 @@
 // See COPYRIGHT and LICENSE files for more details.
 //++
 
-import { Injectable } from '@angular/core';
```
Comment (DmitryBash) [inline frontend/src/app/features/user-preferences/state/user-preferences.store.ts:29]: linter

**Review by oliverguenther — CHANGES_REQUESTED** [review]
The change is correctly saved, but if I remove all days and click save, they pop back in the UI until I reload the page. This only happens when I remove all days.

Peek 2022-08-11 09-27

Also a feature spec addition to spec/features/notifications/settings/reminder_email_settings_spec.rb would be nice, I can help with that if you need guidance

**Comment by DmitryBash** [comment]
@oliverguenther about the feature spec, maybe you mentioned the wrong file? I've added some additional spec in spec/features/notifications/settings/workdays_settings_spec.rb . Correct me if I'm wrong

**Review by oliverguenther — APPROVED** [review]
Ah yes, I was not aware of the separate spec. That's the correct location 👍

I can't reproduce the above issue anymore. LGTM

**Comment by oliverguenther** [comment]
Related WP: https://community.openproject.org/wp/43158

**Pull request merged**